### PR TITLE
Add logoff_time and duration to logon_sessions

### PR DESCRIPTION
**Pull request by directionless**
add logoff_time and duration columns to windows logon_sessions

Fixes: #5556

Note: I did not test this

### CODE REVIEW

File: osquery/tables/system/windows/logon_sessions.cpp [hunk 1]
```diff
@@ -73,7 +73,21 @@ QueryData queryLogonSessions(QueryContext& context) {
       if (sid) {
         LocalFree(sid);
       }
-      r["logon_time"] = BIGINT(longIntToUnixtime(session_data->LogonTime));
+
+      auto logon_time = longIntToUnixtime(session_data->LogonTime);
+      auto logoff_time = longIntToUnixtime(session_data->LogffTime);
```
Comment (directionless) [inline osquery/tables/system/windows/logon_sessions.cpp:78]:
```suggestion
      auto logoff_time = longIntToUnixtime(session_data->LogoffTime);
```

**Comment by theopolis** [comment]
Anything you need help with to test this? I think we should wait for you to double-check and at least smoke test this.

**Comment by directionless** [comment]
> Anything you need help with to test this? I think we should wait for you to double-check and at least smoke test this.

Reasonable. I'll smoke test it on a windows box

**Comment by directionless** [comment]
Excellent callout... While the code was maybe fine, the underlying assumptions are not.

`LsaGetLogonSessionData` is documented as enumerating the _active_ sessions, which testing on my windows laptop never includes useful data in the logoff time. It's either less than 0, or 2^63 - 1.

**Pull request closed without merging**